.gitignore

```diff
@@ -5,8 +5,8 @@ mstpd
 
 /bridge-stp
 /bridge-stp.tmp
-/lib/ifupdown.sh
```

Comment:
My thinking here was that lib/ is typically used either to store library dependencies required by the project or to store source files used to build a library provided by the project, so having shell scripts, man pages, etc in there doesn't line up with the source directory layout I've come to expect from most projects, and I was a bit confused when I first looked for those scripts in the source and found them under the lib/ directory.  I used utils/ as the new name because most of the files in this directory seemed to be derived from brctl-utils, but that choice of a new name was rather arbitrary - basically anything other than lib/ would have eliminated my confusion here.

However, I don't feel particularly strongly about this ... If you don't agree that lib/ is a confusing name for this directory, then I'm fine with dropping this commit.